float64[7] q_cur
float64[7] dot_q_curr
float64[7] ddot_q_curr
float64[700] H_stack
float64[3] ee_velocity
int32 count
int32 l